src/components/ui/Select.js

```diff
@@ -0,0 +1,97 @@
+import Select, { createFilter } from 'react-select'
+import { FixedSizeList } from 'react-window'
+import styled from 'styled-components/macro'
+
+const LIST_ITEM_HEIGHT = 46
+
+const StyledSelect = styled(Select)`
+  font-size: 18px;
```

Comment:
Is it supposed to be this big? The component seems to have a bigger font size and height than the other ones like the input box. Is this intentional?

Reply:
^ good question.

Reply:
Not really intentional, was just following the figma styling. Should I match it to the input box?

Reply:
Yes -- components are bigger on mobile than on desktop, but we'll handle that later. Also, can you style the text that comes up when searching to be same as the input box, too? Thanks.

Reply:
Wait actually @SirajChokshi can you clarify, is there a difference between the heights of the select and the input?